app/validators/sushi_validator.rb

```diff
@@ -1,23 +1,21 @@
 class SushiValidator < ActiveModel::EachValidator
 
   def validate_each(record, _schema, _value)
-    # unless  record.is_valid_sushi? 
     valid = record.validate_sushi
-    unless valid == true
+    unless valid == true or valid == []
```

Comment:
Valid can be either true or empty array?

Reply:
No. This was part of code to get some tests passing. There is a lot more to come on this, but we agreed that we would deal with the issues with the tests separately, since they exist also in R5.

Reply:
I believe we have only observed it as an empty array or an a array with stuff in it.
Part of refactoring/re-invigorating the test suite will be to see if it can ever have a value of `true`